closed/Intel/code/retinanet/pytorch-cpu/run_docker_download_images.sh

```diff
@@ -0,0 +1,53 @@
+if [[ -z "$1"  ||  -z "$2" ||  -z "$3"  ]] ; then
```

Comment:
why do we need to have another file for calibration? 
"--dataset-path /workspace/data/openimages-calibration" could be part of the commands.